components/03-organisms/menu/primary-nav/_yds-primary-nav.scss

```diff
@@ -31,31 +33,37 @@
   @media (min-width: tokens.$break-mobile) {
     position: relative;
     padding: var(--size-spacing-5);
-    border-bottom: var(--border-thickness-1) solid transparent;
-    border-top: var(--border-thickness-1) solid transparent;
-    border-right: var(--border-thickness-1) solid transparent;
-    border-left: var(--border-thickness-1) solid transparent;
 
-    &[aria-expanded='true'] {
-      z-index: 1;
-      border-color: var(--color-navigation-border);
-      border-bottom-color: var(--color-background);
+    [data-menu-variation='basic'] & {
+      padding-top: var(--size-spacing-2);
+    }
+
+    [data-menu-variation='mega'] & {
+      border: var(--border-thickness-1) solid transparent;
 
       .primary-nav__item--level-0:first-child & {
-        padding-left: var(--size-spacing-5);
+        padding-left: 0;
       }
 
       .primary-nav__item--level-0:last-child & {
-        padding-right: var(--size-spacing-5);
+        padding-right: 0;
       }
     }
 
-    .primary-nav__item--level-0:first-child & {
-      padding-left: 0;
-    }
+    &[aria-expanded='true'] {
+      z-index: 1;
+      border-color: var(--color-navigation-border);
+      border-bottom-color: var(--color-background);
 
-    .primary-nav__item--level-0:last-child & {
-      padding-right: 0;
+      [data-menu-variation='mega'] & {
+        .primary-nav__item--level-0:first-child & {
+          padding-left: var(--size-spacing-5);
+        }
+
+        .primary-nav__item--level-0:last-child & {
+          padding-right: var(--size-spacing-5);
+        }
```

Comment:
I know this is how I wrote it before, but if you wanted to update any of these instances where we're doing `.specific-class-name:first-child &` to just `:first-child > &` I think that would be good - but not totally necessary.

Reply:
This is also in place now. :+1: